modules/osdk-bundle-validate-run.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-bundle-validate.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-bundle-validate-run_{context}"]
+= Running the bundle validate command
+
+The default test runs every time you enter the `bundle validate` command. If you want to run optional tests using the `--select-optional` flag, those test run in addition to the default test.
+
+.Prerequisites
+
+* Operator project generated by using the Operator SDK
+
+.Procedure
+
+. If you want to run the default test against a local Operator bundle directory, enter the following command:
++
+[source,terminal]
+----
+$ operator-sdk bundle validate <bundle_dir>
+----
+
+. If you want to run the default test against a remote Operator bundle image, enter the following command:
++
+[source,terminal]
+----
+$ operator-sdk bundle validate \
+  <bundle_registry>/<bundle_image_name>:<tag>
+----
++
+where:
+
+<bundle_registry>:: Specifies the registry where the bundle is hosted, such as `quay.io/example`.
+<bundle_image_name>:: Specifies the name of the bundle image, such as `memcached-operator`.
+<tag>:: Specifies the tag of the bundle image, such as `v1.22.0`.
++
+[NOTE]
+====
+If you want to test an Operator bundle image, you must host your image in a remote registry. The Operator SDK pulls the image and builds it locally before running tests. The `bundle validate` command does not support testing local bundle images.
```

Comment:
Is this referring to the case where I can build an image and it is in my docker image list but not pushed? If so, then this is correct. When I think of local bundle images I think of the bundle directory :) but I think that's because I know that you have to push the image.